### PR TITLE
Sort mandataris publication status selector options

### PR DESCRIPTION
**Pull request by ruben-bottu**
## Description

We want to be able to show the mandataris publication statuses in a consistent order, regardless of the order in which they were inserted into the database. This can be achieved by adding an extra sh:order triple to the concepts in the publication status migration. These triples will then be used in the frontend for manual sorting.

## How to test

- Go to any mandate that is not in the "Bekrachtigd" state.
- Click open the "Publicatie status" selector and check that the statuses are in this order:
    - Draft
    - Effectief
    - Bekrachtigd

image

## Links to other PR's

- https://github.com/lblod/app-lokaal-mandatenbeheer/pull/118

### CODE REVIEW

File: app/models/mandataris-publication-status-code.js [hunk 1]
```diff
@@ -4,6 +4,7 @@ import { MANDATARIS_BEKRACHTIGD_STATE } from 'frontend-lmb/utils/well-known-uris
 export default class MandatarisPublicationStatusCodeModel extends Model {
   @attr uri;
   @attr label;
+  @attr('number') order;
```
Comment (JonasVanHoof) [inline app/models/mandataris-publication-status-code.js:7]: Do we need a default?

**Review by JonasVanHoof — APPROVED** [review]
Works! Also ran the frontend 👍🏻

File: app/models/mandataris-publication-status-code.js [hunk 2]
```diff
@@ -12,5 +13,6 @@ export default class MandatarisPublicationStatusCodeModel extends Model {
   rdfaBindings = {
     class: 'http://www.w3.org/2004/02/skos/core#Concept',
     label: 'http://www.w3.org/2004/02/skos/core#prefLabel',
+    order: 'http://www.w3.org/ns/shacl#order',
```
[the remark this answers is not in the document]
Comment (ruben-bottu) [reply app/models/mandataris-publication-status-code.js:16]: We only use it for concept scheme models/resources, but not sure why

[the remark this answers is not in the document]
Comment (JonasVanHoof) [reply app/models/mandataris-publication-status-code.js]: Nice that you removed it!

**Review by JonasVanHoof — APPROVED** [review]
Can be merged! After the backend ;p

**Pull request merged**